packages/engine/Source/Scene/ITwinData.js

```diff
@@ -160,6 +160,7 @@ ITwinData.createDataSourceForRealityDataId = async function (
   realityDataId,
   type,
   rootDocument,
+  // TODO: is there a nice-ish way to thread through custom access tokens per asset?
```

Comment:
I think we agreed this doesn't make sense for the time being given the series of calls currently needed, right? Let's remove this comment for now.

Perhaps when there is a single API endpoint for various datatypes, we can consider adding this option.